### ISSUE TITLE
Processor model not detected on PowerPC G5

### ISSUE THREAD

**Issue by MingcongBai** [dylanaraps/neofetch#526]
## Description

When running `neofetch` on a G5, the processor model is not detected correctly from `/proc/cpuinfo`, as seen in the screenshot below.

![2016-12-13 08-14-58](https://cloud.githubusercontent.com/assets/5006263/21143423/519aa496-c10c-11e6-8fad-3d8aa11bbfcc.png)

However, the CPU model **is** listed in `/proc/cpuinfo` (I believe this is the way `neofetch` gets the CPU model from the `get_cpu` function), in the `cpu: ` field, but not the usual `model name: ` field for Intel processors. Here is a sample output of `cat /proc/cpuinfo` on my G5.

```
root@G5-AOSC [ / ] # cat /proc/cpuinfo 
processor	: 0
cpu		: PPC970MP, altivec supported
clock		: 1250.000000MHz
revision	: 1.1 (pvr 0044 0101)

processor	: 1
cpu		: PPC970MP, altivec supported
clock		: 1250.000000MHz
revision	: 1.1 (pvr 0044 0101)

processor	: 2
cpu		: PPC970MP, altivec supported
clock		: 1250.000000MHz
revision	: 1.1 (pvr 0044 0101)

processor	: 3
cpu		: PPC970MP, altivec supported
clock		: 1250.000000MHz
revision	: 1.1 (pvr 0044 0101)

timebase	: 33333333
platform	: PowerMac
model		: PowerMac11,2
machine		: PowerMac11,2
motherboard	: PowerMac11,2 MacRISC4 Power Macintosh 
detected as	: 337 (PowerMac G5 Dual Core)
pmac flags	: 00000000
L2 cache	: 1024K unified
pmac-generation	: NewWorld
```

#### Neofetch version

Version 2.0.2.

## Verbose log

[neofetchlog.txt](https://github.com/dylanaraps/neofetch/files/649135/neofetchlog.txt)

**Comment by MingcongBai** [comment]
Just a side note, this is also reproducible on a YeeLoong 8089D laptop.

**Comment by konimex** [comment]
I believe the CPU is `PPC970MP` and not the `PowerMac G5` itself?

Also, what's the output of `/proc/cpuinfo` in your Yeeloong?

I don't know if using `lscpu` is more accurate or not, but what is the output of `lscpu` on both of your devices? (assuming `util-linux` package exists, but I think it should exist since it is a system package)

**Comment by MingcongBai** [comment]
> I believe the CPU is PPC970MP and not the PowerMac G5 itself?

To be more exact: `PPC970MP, altivec supported`.

> Also, what's the output of /proc/cpuinfo in your Yeeloong?

This I couldn't provide at the moment, I would need to re-deploy AOSC on it - just ignore it for now, but as far as I can recall, on a 8089D, a processor model is detected in `/proc/cpuinfo`, but not from `lscpu`...

> I don't know if using lscpu is more accurate or not, but what is the output of lscpu on both of your devices? (assuming util-linux package exists, but I think it should exist since it is a system package)

The case of 8089D should answer this question on its own... `lscpu` could sometimes omit processor model output.

**Comment by dylanaraps** [comment]
Does this command print the correct information on your G5?

```sh
awk -F ': | @' '/^model name|^Processor|^cpu  / {printf $2; exit}' /proc/cpuinfo
```

**Comment by konimex** [comment]
Looking from the thread you linked, I think the accurate CPU is the numbered one, as "PowerPC G5" is ~~marketing buzzword~~[what Apple calls them](https://en.wikipedia.org/wiki/PowerPC_970).

**Comment by dylanaraps** [comment]
This command should cover all bases then right?

```sh
awk -F ': | @' '/^model name|^Processor|^cpu  / {printf $2; exit}' /proc/cpuinfo
```

**Comment by konimex** [comment]
@MingcongBai, does #529 fix your issue?

**Comment by konimex** [comment]
I've merged #529 as I feel that the PR should fixed the problem. However, if it doesn't fix your problem please reopen this bug.
